VideoLocker/src/main/java/org/edx/mobile/module/analytics/ISegment.java

```diff
@@ -8,6 +8,8 @@
 
 import org.edx.mobile.util.images.ShareUtils;
 
+import java.util.Map;
+
 public interface ISegment {
```

Comment:
I can make a commit specific to that refactoring in this PR. Cuz I like the idea :wink: